README.md

```diff
@@ -1,2 +1,91 @@
 # queries
+
 Hosts reusable log queries which are built into a single queries.json file.
+
+## Query database structure
+
+```yaml
+queries:
+  pip-no-matching-distribution:  # also known as query "id", (sort key)
+    pattern: "ERROR: No matching distribution found for"  # required, can be list!
+    # optionals:
+    name: "PIP failed to find package"  # optional (aka 'reason'), id is used instead
+    files: []  # list of glob patterns, narrows down searching
+    regex: false  # optional, default is false
+    url: ... # str or list[str], issue links to lp, bz,...
+    category: ... # see Categories
+    tags: []  # see Tags
+    suppress-graph: false  # used for elastic-recheck
```

Comment:
"suppress-graph" is still unclear

Reply:
Is for compatibility, i seen used in eleastic-recheck at least once. It is kept in order to allow us to compile configs that can be loaded by it. We can drop it later.